api/src/config/tokenExpiry.js

```diff
@@ -1,7 +1,6 @@
+const aceessTokenExpiry = '15m';
```

Comment:
This is a nitpick, but I almost wonder if we can pull out the value in minutes, and then for the token expiry convert it to string and add a 'm' at the end. So that this value can be used for the cookie expiry as well by multiplying it by 60000 milliseconds per minute. eg:
`const accessTokenExpiryValue = 15;`
`const accessTokenExpiry = accessTokenExpiryValue.toString() + 'm';`
`const accessTokenCookieExpiry = accessTokenExpiryValue * 60000;`
That way we only need to change accessTokenExpiryValue, and not accessTokenExpiry and accessTokenCookieExpiry.

Reply:
Also spelling mistake in access

Reply:
I think I made a serious mistake here cause the accessTokenExpiry should not be equal to its accessTokenCookieExpiry. Even though the accessToken is expired, we still need this to pass it to the backend to generate a new one, so we don't want it to be removed from the cookies section. I'll get it fixed.